services/app/src/components/gameFooter/GameFooterButtons.jsx

```diff
@@ -13,9 +13,18 @@ const GameFooterButtons = ({ skipCurrentPlayerMove }) => {
         className="finish-btn padding-x-m padding-y-s rubik-font" 
         disabled={isAllTurnsMade || finished || !lastTurn} 
         onClick={finishMove}
+        hidden={finished}
       >
         Finish Move
       </button>
+      <button 
+        type="button" 
+        className="unselect-btn padding-x-m padding-y-s rubik-font" 
+        onClick={clearBoard}
```

Comment:
If I understand correctly, the button clears the board for all the players, however, when I unselect my move, I would expect that only my move was reverted.